### DISCUSSION TITLE
Rotational axis of tilt boundary

### DISCUSSION THREAD

**Discussion by da-levy** [mtex-toolbox/mtex#824] [Ask Anything]
Hello,

I want to find the rotational axes of points across a subgrain boundary. If I plot a pole figure of all low-index planes within the subgrain domain, I see a clear rotation of grains around a single point (the rotation axis; attached). Is there a way to plot the rotational axes themselves (example attached)?

I found the axis() function, which seems to be the way to do that. However, the result from axis() does not correspond to the rotational axis determined through visual inspection of the pole figure. 

My input is: axis(ebsd_subset.orientations)
I realize this is probably not the correct input for my purpose. I am guessing I should be inputting an array of points from either side of the subgrain boundary (axis(o1, o2). I am not really sure, though.

Figure 1: My plot of low index planes. Rotational axes from axis() are a blue cluster at bottom right. Clearly, these points should cluster slightly to the up and left.
![16_SB_05_x5_cleanpolefig_dispersion_withRotationalAxis](https://user-images.githubusercontent.com/68292240/109681665-9008a900-7b32-11eb-8681-738f9d94deff.png)

Figure 2: The below plot if my goal. This figure is from Steve Reddy et al. (2007)
![Screenshot (31)](https://user-images.githubusercontent.com/68292240/109682282-2210b180-7b33-11eb-9ea7-7e6582849dfe.png)

Thanks in advance,
Drew

**Comment by kilir** [comment]
Hi,
it's still a bit obscure to me what you want as input. If `ebsd_subset.orientations` are all orientations within a grains and not only those along a low angle boundary and you'd liek to repeat Reddy etal., it's probably (2).

1) Using `'axis(o1,o2)'` would give a misorientations axis in specimen coordinates - this would make sense in many cases if o1 and o2 are pairs or orientations separated by a low angle boundary. Have a look here on how to use it: https://mtex-toolbox.github.io/TiltAndTwistBoundaries.html#6

2) Your plot looks like somewhat showing the orientation spread within a grain (overall spread is small) and if you want to find a specimen/crystal direction with the smallest spread, use `f = fibre.fit(ebsd_subset.orientations)` where `f.h` will be this direction in crystal coordinates, f.r in `specimen coordinates`. 

Hope that helps.

Cheers,
Rüdiger
